src/main/java/uk/gov/hmcts/reform/sendletter/entity/BasicLetterInfo.java

```diff
@@ -22,7 +23,8 @@ public BasicLetterInfo(
         String type,
         String encryptionKeyFingerprint,
         LocalDateTime createdAt,
-        LocalDateTime sentToPrintAt
+        LocalDateTime sentToPrintAt,
+        LocalDateTime printedAt
```

Comment:
this will be useful I assume supplier sends this information when we reconcile reports ?

Reply:
yeah, it's in DB, just adding it to the 'output' model